app/views/admin/users/index/_user_actions_dropdown.html.erb

```diff
@@ -47,7 +47,20 @@
                 <li><button class="c-btn align-left w-100 c-btn--icon-left"><%= crayons_icon_tag("email", aria_hidden: true, class: "c-btn__icon") %>Send an email</button></li>
                 <li><button class="c-btn align-left w-100 c-btn--icon-left"><%= crayons_icon_tag("note", aria_hidden: true, class: "c-btn__icon") %>Write a note</button><hr /></li>
                 <li><button class="c-btn align-left w-100 c-btn--icon-left"><%= crayons_icon_tag("cloud-off", aria_hidden: true, class: "c-btn__icon") %>Unpublish all posts</button></li>
-                <li><button class="c-btn align-left w-100 c-btn--icon-left"><%= crayons_icon_tag("warning", aria_hidden: true, class: "c-btn__icon") %>Banish user</button></li>
+                <% if !user.banished? %>
```

Comment:
The conditional rendering of the "Banish user" button wasn't in the designs, but I like the idea of hiding the button if the user has already been banished (since the action has already been taken and it can only be taken once). This button is also still visible and available post-banishment of a user on their Member show page.

This all aside, I'm happy to revert this change if preferred or if it's bad for accessibility.  

cc @Ridhwana @aitchiss

Reply:
I think that's a really useful change 👍